packages/core/src/scp-cf/env-destination-accessor.ts

```diff
@@ -2,9 +2,9 @@
 
 import { createLogger } from '@sap-cloud-sdk/util';
 import {
-  addProxyConfigurationInternet,
+  proxyStrategy,
   ProxyStrategy,
-  proxyStrategy
+  addProxyConfigurationInternet
```

Comment:
run linter for alphabetic ordering.

Reply:
It seems that our linter does not do that ;)